map/src/main/res/layout/activity_map.xml

```diff
@@ -18,6 +18,23 @@
         app:navGraph="@navigation/map_navigation_graph"
         tools:background="#fab" />
 
+
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="50dp"
```

Comment:
why set manual height. also this is hardcoded. is this shared on other screens? im guessing yes so it should be in `dimens`. dont be lazy :P

Reply:
Yea didn't think about the accessibility and other stuff that might increase the text size and they might cutoff if height is absolute. Nice catch Andrew. Nice catch. Updated